dashboard/app/main.html

```diff
@@ -111,7 +111,8 @@
 
 	{{range $ns := $.BugNamespaces}}
 		<br>
-		<h2 id="{{$ns.Name}}">{{$ns.Caption}}</h2>
+		<a name="{{$ns.Name}}"></a>
```

Comment:
Do we still need this anchor?

Reply:
You're totally right. This was redundant to the id=... Now fixed! :)